actionmailbox/config/routes.rb

```diff
@@ -16,7 +16,9 @@
 
   # TODO: Should these be mounted within the engine only?
   scope "rails/conductor/action_mailbox/", module: "rails/conductor/action_mailbox" do
-    resources :inbound_emails, as: :rails_conductor_inbound_emails, only: %i[index new show create]
+    inbound_emails_default_actions = %i[index show create]
+    inbound_emails_default_actions << :new unless Rails.application.config.api_only
```

Comment:
I like this PR but I don't like this change. Can we change the validation to not take in consideration api_only? Let's just check if the values are inside `[:index, :create, :new, :show, :update, :destroy, :edit]` always.

Reply:
Thanks for the review, I've actioned this.